templates/hooks/certificate_authority_activation/sdk_create_post_set_output.go.tpl

```diff
@@ -1,5 +1,5 @@
     if certificateSecret != "" {
-        err = rm.writeCertificateChainToSecret(ctx, certificateSecret, certificateChainSecret, desired.ko.ObjectMeta)
+        err = rm.writeCertificateChainToSecret(ctx, certificateSecret, certificateChainSecret, desired.ko.ObjectMeta.GetNamespace(), desired.ko.Spec.CompleteCertificateChainOutput)
```

Comment:
`ObjectMeta` is an embedded field, so this could be condensed to:

```suggestion
        err = rm.writeCertificateChainToSecret(ctx, certificateSecret, certificateChainSecret, desired.ko.GetNamespace(), desired.ko.Spec.CompleteCertificateChainOutput)
```